Admin/AbstractAdmin.php

```diff
@@ -2838,22 +2838,12 @@ public function getDashboardActions()
     }
 
     /**
-     * Setting to true will enable mosaic button for
-     * the admin screen.
-     * Setting to false will hide mosaic button for
+     * Setting to true will enable mosaic button for the admin screen. Setting to false will hide mosaic button for
      * the admin screen.
      */
-    public function setIsShowMosaicButton($isShow)
+    public function setIsShowMosaicButton($isShown)
```

Comment:
@greg0ire I don't want to argue with you, but I try to write clear code. For example, instead of write `turn($value)` function, I prefer create to methods like `turnOn()` and `turnOff`, for me it's more simple. If I want to `hide` something, I will look for method `show` with opposite argument in the last thing. If you want to optimize huge `Admin` class, I think, we need to highlight main parts, add plan how to separate it, and create a global strategy to fulfill it and save BC and etc.